pkg/scheduler/core/generic_scheduler.go

```diff
@@ -134,6 +134,10 @@ type ScheduleAlgorithm interface {
 	// for cluster autoscaler integration.
 	// TODO(ahg-g): remove this once CA migrates to creating a Framework instead of a full scheduler.
 	PredicateMetadataProducer() predicates.MetadataProducer
+	// Snapshot snapshots scheduler cache and node infos. This is needed
+	// for cluster autoscaler integration.
+	// TODO(ahg-g): remove this once CA migrates to creating a Framework instead of a full scheduler.
```

Comment:
create an issue and put it in the TODO instead.

Reply:
done.